app/src/main/java/com/gh4a/utils/HttpImageGetter.java

```diff
@@ -342,8 +340,7 @@ public HttpImageGetter(Context context) {
         WindowManager wm = (WindowManager) context.getSystemService(Context.WINDOW_SERVICE);
         final Point size = new Point();
         wm.getDefaultDisplay().getSize(size);
-        mMaxWidth = size.x;
-        mMaxHeight = size.y;
+        mMaxWidth = size.x - UiUtils.dpToPixels(context, 48);  // take horizontal padding into account
```

Comment:
Do not hardcode sizes, for two reasons:
- the image getter doesn't know about the view it's used in
- different views might use different paddings

If anything, pass in the actual view's usable size, but since the point of scaling down beyond the screen size is just to save memory, I'm not sure what the point of that is?

Reply:
> I'm not sure what the point of that is?

Well, the point of scaling is that if you don't do it, an image wider than the screen will be rendered at full size and the parts that don't fit in the screen (or better, in the TextView) will be cut off.
I totally agree that this solution is far from ideal and that we should use the TextView size instead. The problem is that doing this would require a quite big refactor since the TextView is stored in `ObjectInfo` class... we should at the very least move the scaling logic there.

I've used 48dp since it's the highest TextView padding I could find in the app (the repository README TextView). It's not a big deal if we remove this for now, the only downside is that the border of some wide images might get cut off a bit.

Reply:
Reverted